rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
```

Comment:
- [ ] Kindly make sure the Getters and Setters for @book and @person instance variables be present 👇🏻 


```ruby
     attr_accessor :date, :book, :person
```